cardano-cli/src/Cardano/CLI/Run/Debug/LogEpochState.hs

```diff
@@ -0,0 +1,42 @@
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE NamedFieldPuns #-}
+
+module Cardano.CLI.Run.Debug.LogEpochState
+  ( runLogEpochStateCmd
+  ) where
+
+import           Cardano.Api
+import qualified Cardano.Api as Api
+
+import           Cardano.CLI.Commands.Debug.LogEpochState
+import           Cardano.CLI.Orphans ()
+
+import           Data.Aeson
```

Comment:
This module is small now, but can you import `Aeson` as qualified or at least explicit?